created file adding volume component and details of openshift component (#132)
devfile/docs: docs/modules/user-guide/partials/proc_adding-a-volume-component-to-a-devfile.adoc

[id="proc_adding-a-volume-component-to-a-devfile_{context}"]
= Adding a volume component to a devfile

[role="_abstract"]

> You can use a volume component in order to share files among container components, making it easier to collaborate with other teams during the development process.

Reviewer: IBM SG: "Use "to" in most cases to avoid wordiness."

Reviewer: Hi, @Jake Berger. I was revisiting this PR and realized I missed something important during my peer review. The content talks about adding a component to a devfile, but omits the effect of this action in OpenShift, and then describes the benefit. I'd recommend something more like:
```suggestion
To create a <name the type of volume> in your shared <location of volume>, you can add a volume component to your devfile. This <type of volume> makes it easier for teams to share files and collaborate with each other during the development process.
```
WDYT?

Reviewer: A similar approach might also apply to docs/modules/user-guide/partials/proc_adding-a-kubernetes-or-openshift-component-to-a-devfile.adoc

Author: Devs, I think Rolfe's suggestion is good, but I honestly wouldn't be able to say what type of component it is. Could any of you give an idea to expand our abstract more?

> You can use a volume component in order to share files among container components, making it easier to collaborate with other teams during the development process.

Reviewer 2: ```suggestion
You can use a volume component to share files among container components and collaborate easily with other teams during the development process.
```

.Prerequisites

* xref:adding-schema-version-to-a-devfile.adoc[]
* xref:adding-a-name-to-a-devfile.adoc[]
* xref:adding-components-to-a-devfile.adoc[]

.Procedure

* Define a `volume` component in your devfile.
+
.A minimal `volume` component
[source,yaml]
----
schemaVersion: 2.1.0
metadata:
    name: mydevfile
components:
  - name: mydevfile
    volume:
      size: 2Gi
----

> +

Reviewer 2: Suggestion: If you have a verification step, please include one. Also, if there is no verification step, add a sentence after the procedure to close the loop of the procedure section. That way, the section looks complete.

[role="_additional-resources"]

> .Additional resources

Reviewer: This heading seems to be missing from the preview: https://devfile-docs-preview-pr-132.surge.sh/devfile/2.2.0/user-guide/adding-a-volume-component-to-a-devfile.html

Reviewer 2: +1 to Rolfe's comment. @jc-berger PTAL and rectify.
